lib/Model/DevCycleUser.php

```diff
@@ -716,8 +702,91 @@ public function __toString()
      *
      * @return string
      */
-    public function toHeaderValue()
+    public function toHeaderValue(): string
     {
         return json_encode(ObjectSerializer::sanitizeForSerialization($this));
     }
+
+    public static function FromEvaluationContext(EvaluationContext $context): DevCycleUser
```

Comment:
Do we have unit tests for this?

Reply:
not directly - it's on my list to do

Reply:
like for all the edge-cases and error handling? 

Do any of these limitation's / restrictions similarly apply here? https://docs.devcycle.com/sdk/server-side-sdks/node/node-openfeature#context-properties-to-devcycleuser

Reply:
I think we need some form of tests here before we release this. But we can merge this PR ahead of that.

Reply:
Yeah - this isn't intended to be a one and done - just the first step in a major refactor that needs to also be matched on the test harness